Remove Like action from inbox chat messages

ESWAT opened the pull request:
Fixes #10132

### Changes
Removes Like action from inbox chat messages. There is no API endpoint for this action and seems rather useless for private messages anyway,

![screen shot 2018-03-25 at 15 16 56](https://user-images.githubusercontent.com/65468/37879027-99876b9c-303f-11e8-8f7a-1755a3afd1ba.png)

----
UUID: ffee6b77-0bf8-422a-a65c-c20de17f2b32

negue commented on website/client/components/chat/chatCard.vue:
```diff
@@ -222,6 +222,7 @@ export default {
   },
   methods: {
     async like () {
+      console.log(this);
```
could you remove the `console.log` ?

ESWAT replied:
Removed (squashed my commits so the original doesn’t have this mistake).

ESWAT committed:
Remove Like action from inbox chat messages

There is no API endpoint for this action and seems rather useless for private messages anyway
